zellij-server/src/wasm_vm.rs

```diff
@@ -54,6 +54,7 @@ pub(crate) struct PluginEnv {
     pub senders: ThreadSenders,
     pub wasi_env: WasiEnv,
     pub subscriptions: Arc<Mutex<HashSet<EventType>>>,
+    pub _allow_exec_host_cmd: bool,
```

Comment:
The leading `_` is good here! If you want to be even more explicit about this being temporary, if might be nice to leave a comment here :) Maybe starting like: `// FIXME: ...` so it's easy to find later :)